File: AOJ/ITP1/python/ITP1_10_C_Standard_Deviation.py
import math
from statistics import pstdev
while int(input()):
    score = list(map(int, input().split()))
    """
    M = sum(score) / len(score)
    SD = math.sqrt(sum([(n - M) ** 2 for n in score]) / len(score))
    print("{:.4f}".format(SD))
    """
    print("{:.4f}".format(pstdev(score)))

File: AOJ/ITP1/csharp/ITP1_10_C_Standard_Deviation.cs
using System;
using System.Linq;

class P
{
    static void Main()
    {
        while (Console.ReadLine() != "0")
        {
            var a = Console.ReadLine().Split().Select(int.Parse);
            Console.WriteLine(Math.Sqrt(a.Select(x => Math.Pow(x - a.Average(), 2)).Sum() / a.Count()));
        }
    }
}
